src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    private final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int randomColorIndex = random.nextInt(Color.values().length);
+        return Color.values()[randomColorIndex].name();
```

Comment:
`Color.values()` call result can be saved to a variable to avoid multiple calls

Reply:
Made Color.values() a separate variable.